### ISSUE TITLE
Cannot duplicate docker / Tensorflow example in documentation

### ISSUE THREAD

**Issue by chrisreidy** [apptainer/singularity#393]
Hi
I could not replicate the line:
singularity import tensorflow.img docker://tensorflow/tensorflow:latest
or anything to do with docker://tensorflow/tensorflow:latest without getting:
Error getting tags using url https://registry-1.docker.io/v2/tensorflow/tensorflow/tags/list

Vanessa said that was fixed more recently so I just pulled the latest singularity and get:
singularity create -s 4000 docker-tf.img
Creating a new image with a maximum size of 4000MiB...
Executing image create helper
/cm/shared/apps/singularity/2.2/libexec/singularity/simage: symbol lookup error: /cm/shared/apps/singularity/2.2/libexec/singularity/simage: undefined symbol: singularity_image_bind

So I reinstalled the 2.2 version and the 'create' functioned normally

**Comment by gmkurtzer** [comment]
@vsoch, is this related to #392?

**Comment by vsoch** [comment]
For the first error about the tensorflow registry - the answer is likely yes. There seem to have been some recent changes in the manifest versions, and the PR we just merged address those. 

The issue @chrisreidy is now posting looks like it's related to the `create` function, I also tested locally, and the same command seems to work ok:

```bash
vanessa@vanessa-ThinkPad-T460s:~/Desktop$ sudo singularity import tensorflow.img docker://tensorflow/tensorflow:latest
Cache folder set to /home/vanessa/.singularity/docker
Downloading layer sha256:56eb14001cebec19f2255d95e125c9f5199c9e1d97dd708e1f3ebda3d32e5da7
Extracting /home/vanessa/.singularity/docker/sha256:56eb14001cebec19f2255d95e125c9f5199c9e1d97dd708e1f3ebda3d32e5da7.tar.gz
Downloading layer sha256:7ff49c327d838cf14f7db33fa44f6057b7209298e9c03369257485a085e231df
Extracting /home/vanessa/.singularity/docker/sha256:7ff49c327d838cf14f7db33fa44f6057b7209298e9c03369257485a085e231df.tar.gz
Downloading layer sha256:6e532f87f96dd5821006d02e65e7d4729a4e6957a34c3f4ec72046e221eb7c52
Extracting /home/vanessa/.singularity/docker/sha256:6e532f87f96dd5821006d02e65e7d4729a4e6957a34c3f4ec72046e221eb7c52.tar.gz
Downloading layer sha256:3ce63537e70c2c250fbc41b5f04bfb31f445be4034effc4b4c513bf8899dfa0a
Extracting /home/vanessa/.singularity/docker/sha256:3ce63537e70c2c250fbc41b5f04bfb31f445be4034effc4b4c513bf8899dfa0a.tar.gz
Downloading layer sha256:52f5a652594c5e16bad1da1017927faa4d85a13b9c9b578d87c7941355d4360e
Extracting /home/vanessa/.singularity/docker/sha256:52f5a652594c5e16bad1da1017927faa4d85a13b9c9b578d87c7941355d4360e.tar.gz
Downloading layer sha256:f2deccc66bac10ed9800309edf6b6c4f726e4949af80ff5c9a58ad345c2b8f52
Extracting /home/vanessa/.singularity/docker/sha256:f2deccc66bac10ed9800309edf6b6c4f726e4949af80ff5c9a58ad345c2b8f52.tar.gz
Downloading layer sha256:a6d3787029a600c07fc457d8051ee3402e677f2fd99e21a7ecc7a3501c62bab9
Extracting /home/vanessa/.singularity/docker/sha256:a6d3787029a600c07fc457d8051ee3402e677f2fd99e21a7ecc7a3501c62bab9.tar.gz
Downloading layer sha256:de0e1648fa8ef8c9ea79292b75328413ffac4b7b737215c411b3cb2151b8aeea
Extracting /home/vanessa/.singularity/docker/sha256:de0e1648fa8ef8c9ea79292b75328413ffac4b7b737215c411b3cb2151b8aeea.tar.gz
Downloading layer sha256:9b1d17ee0de50d9d408ea5242c8290bfe11838065c18c5c7b2947b3b19b71074
Extracting /home/vanessa/.singularity/docker/sha256:9b1d17ee0de50d9d408ea5242c8290bfe11838065c18c5c7b2947b3b19b71074.tar.gz
Downloading layer sha256:ad112d680f8c599c46fcf6864472fcd2379b189f266e387fb666588645c58727
Extracting /home/vanessa/.singularity/docker/sha256:ad112d680f8c599c46fcf6864472fcd2379b189f266e387fb666588645c58727.tar.gz
Downloading layer sha256:1cbaa67245bedb51568ea2ddea4a37baff6764f77a05051c218229cf0c5a30aa
Extracting /home/vanessa/.singularity/docker/sha256:1cbaa67245bedb51568ea2ddea4a37baff6764f77a05051c218229cf0c5a30aa.tar.gz
Downloading layer sha256:65f3587f2637c17b30887fb0d5dbfad2f10e063a72239d840b015528fd5923cd
Extracting /home/vanessa/.singularity/docker/sha256:65f3587f2637c17b30887fb0d5dbfad2f10e063a72239d840b015528fd5923cd.tar.gz
Bootstrap initialization
No bootstrap definition passed, updating container
Executing Prebootstrap module
Executing Postbootstrap module
Done.
```

@chrisreidy I think what likely is happening is that you have an old library file in your install location. I would try completely deleting/removing that folder, reinstalling from master, and trying again. For example, if you set `--prefix=/cm/shared/apps/singularity/2.2` then you would want to delete `/cm/shared/apps/singularity/2.2/libexec/singularity` and there is also some stuff in `etc`:

      rm -rf /cm/shared/apps/singularity/2.2/libexec/singularity
      rm -rf /cm/shared/apps/singularity/2.2/etc/singularity

Give that a go - tell us if you get the same library error!

**Comment by chrisreidy** [comment]
Two problems fixed!  I just erased everything after 2.2 and ran "make install" again - problem solved.  And now with the more recent singularity my docker import is working.  Perhaps I should not push my luck and just go home. Thanks for the prompt reply

**Comment by vsoch** [comment]
Haha, excellent!! I would say it's always good to go home regardless, because that's where a comforting place and dinner can be found :)
